header-headline-mobile

adarachel opened the pull request:
### Setting up Feature branch, Linters & Creating a Pull Request:
- Added HTML & CSS Linters during Project Set Up(Best Practice)
- Added .github/workflows folder and included linters.yml file. 
- Created a feature branch called 'Set-Up' and initiated a pull request.
- Added .gitignore file.
### Building the Header/Toolbar section & Headline section:
- Added 'index.html' file, containing HTML code required to build the mobile toolbar and headline sections.
- Added 'styles.css' file, containing style for the HTML code.
- Implemented 'Flexbox' required to lay out the elements on the 2 sections.
- Implemented button interactions such as hover, active.
-  Added descriptive commit messages and descriptive README file.
- Commented a short summary of overall changes made to pull request.
- Made sure there were no Linter errors.

adarachel committed:
Added .github/workflows folder

adarachel committed:
Project Set Up

adarachel committed:
HTML & CSS Files

Added HTML & CSS file containing code for the toolbar/header section and the headline section for the mobile personal portfolio website project.

adarachel committed:
Fixed stylelint errors

adarachel committed:
Update styles.css

adarachel committed:
Update styles.css

adarachel committed:
Update styles.css

adarachel committed:
Create mit.md

adarachel committed:
README.md

isaacpitwa commented on index.html:
```diff
@@ -0,0 +1,53 @@
+    <!--HEADLINE-->
+    <section class="headline-section">
+        <div class="container-flex">
+            <h1 class="headline-title">Hey there. I'm Rachel</h1>
+            <h1><span>I'm a software developer</span></h1>
```
- [ ] it is more useful to use a single `h1` tag for every single webpage for semantic improvement. kindly change the subtitle to a `p`.

isaacpitwa commented on README.md:
```diff
@@ -1,2 +1,21 @@
+- Linters
+
+# To Run
+To run this project, you need to have a web browser such as Google Chrome, Safari, etc. And you need to clone the repository or download the ZIP files.
```
- [ ][OPTIONAL] Kindly include commands for executing the stated steps for example cloning the repo to best help all kinds of visitors.

isaacpitwa reviewed (changes requested):
Hi @adarachel  🙂 ,

You have done a tremendous job setting up your project. I really like it :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

![Alt Text](https://media.giphy.com/media/qLWdMYX1NYF2g/giphy-downsized-large.gif)


## Highlights 
- All linters are passing  ✅
- Correct use of GitHub flow ✅
- Well documented Repository ✅
-  Clean and readable code base ✅

## Required changes ♻️ 
 Check the comments under the review.
## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Check the comments under the review.





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @isaacpitwa.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

adarachel commented:
Hello reviewer,

I am meant to make the website look exactly like the figma template, aren't
I? If I change that <h1> tag to a <p> tag, the text will become a lot
smaller than the text in the template. How do I proceed please?

Thank you,
Rachel.

On Thu, Sep 22, 2022, 5:21 PM Isaac Pitwa ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @adarachel <https://github.com/adarachel> 🙂 ,
>
> You have done a tremendous job setting up your project. I really like it
> 👍
> There are some issues that you still need to work on to go to the next
> project but you are almost there!
>
> [image: Alt Text]
> <https://camo.githubusercontent.com/0b6134a224332efd68cb15c3a3ff40ac036be75801f5ab22f0905d6b343c8a60/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f714c57644d5958314e594632672f67697068792d646f776e73697a65642d6c617267652e676966>
> Highlights
>
>    - All linters are passing ✅
>    - Correct use of GitHub flow ✅
>    - Well documented Repository ✅
>    - Clean and readable code base ✅
>
> Required changes ♻️
>
> Check the comments under the review.
> Optional suggestions
>
> Every comment with the [OPTIONAL] prefix is not crucial enough to stop the
> approval of this PR. However, I strongly recommend you to take them into
> account as they can make your code better.
>
> Check the comments under the review.
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification @isaacpitwa <https://github.com/isaacpitwa>.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In index.html
> <https://github.com/adarachel/personal-portfolio/pull/1#discussion_r977854490>
> :
>
> > +            <h1 class="headline-title">Hey there. I'm Rachel</h1>
>
> +            <h1><span>I'm a software developer</span></h1>
>
>
>
>    - it is more useful to use a single h1 tag for every single webpage
>    for semantic improvement. kindly change the subtitle to a p.
>
> ------------------------------
>
> In README.md
> <https://github.com/adarachel/personal-portfolio/pull/1#discussion_r977854881>
> :
>
> > @@ -1,2 +1,21 @@
>
> -# personal-portfolio
>
> -Portfolio Website Project
>
> +# Personal Portfolio Website
>
> +Set up and Mobile Version Skeleton.
>
> +
>
> +# Built With
>
> +- HTML
>
> +- CSS
>
> +- GitHub
>
> +- Linters
>
> +
>
> +# To Run
>
> +To run this project, you need to have a web browser such as Google Chrome, Safari, etc. And you need to clone the repository or download the ZIP files.
>
>
>
>    - [ ][OPTIONAL] Kindly include commands for executing the stated steps
>    for example cloning the repo to best help all kinds of visitors.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/adarachel/personal-portfolio/pull/1#pullrequestreview-1117343660>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ53ZBS4KJATU2I36AXWTXDV7SBPHANCNFSM6AAAAAAQSI2D24>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

isaacpitwa commented:
hi @adarachel, you could use style properties like `font-size` and` font-weight` to style a p to look exactly like is in the Figma design and at the same time maintain a higher level of semantics.

adarachel committed:
<h1> to <p> tag

Made required changes from my reviewer. I changed the second <h1> tag in 'index.html' to a <p> tag instead to maintain a "higher level of semantics".

adarachel committed:
Update index.html

adarachel committed:
Merge branch 'set-up' of https://github.com/adarachel/personal-portfolio into set-up

adarachel committed:
Required changes by reviewer

adarachel committed:
Update styles.css

BertrandConxy commented on README.md:
```diff
@@ -1,2 +1,21 @@
-# personal-portfolio
-Portfolio Website Project
+# Personal Portfolio Website
+Set up and Mobile Version Skeleton.
+
+# Built With
+- HTML
+- CSS
+- GitHub
+- Linters
+
+# To Run
+To run this project, you need to have a web browser such as Google Chrome, Safari, etc. And you need to clone the repository or download the ZIP files.
+
```
- [ ] [OPTIONAL] Great job so far 👏 In addition to what you have done, I would suggest you add the `screenshot` of your landing page here in your README so as to give visitors a quick preview of what your portfolio looks like which can increase their interest in knowing you 😉 😎 
- [ ] [OPTIONAL] please elaborate more in the `Getting started` section. This is where you describe procedures to clone and run your project locally. It is better if you add some commands too 😉. This is the most important section for your fellow developers 😃 👍

BertrandConxy reviewed (approved):
Hi @adarachel , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- Great use of comments
- No linter errors
- project requirements are met
- good UI 💯 
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
